Feat: add description and change word (#1)
scroll-tech/scaffold-scroll: README.md

<div align="center">
  <a href="https://scroll.io"><img alt="Scroll" src="https://scroll.io/static/media/Scroll_Banner1.bbb483b6c623f1f3e436be015eba0514.svg"></a>
  <br />
  <br />
</div>

# 🏗 Scaffold-Scroll 📜

<h4 align="center">
  <a href="https://docs.scaffoldeth.io">Documentation</a> |
  <a href="https://scaffoldeth.io">Website</a>
</h4>

> 📜 Scaffold-Scroll is a fork of Scaffold-ETH2 with minimal differences with native support for testnets, aim to help developers get use Scaffold easier on Scroll network.

Reviewer: ```suggestion
📜 Scaffold-Scroll is a fork of Scaffold-ETH2 with minimal differences with native support for Scroll testnets, and aims to help developers use Scaffold-Eth more easily on the Scroll network.
```

Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.

## Deploy Contracts on Scroll Sepolia or L1SLOAD

To deploy contracts to a remote testnet (e.g. Scroll Sepolia), follow the steps below:

> 1. Get Scroll Sepolia ETH from the [Scroll Sepolia Faucet](https://docs.scroll.io/en/user-guide/faucet/) or [Scroll Sepolia thirdweb Faucet](https://thirdweb.com/scroll-sepolia-testnet). Get L1SLOAD testnet token, follow this guide [L1SLOAD GUIDE](https://scrollzkp.notion.site/L1SLOAD-Guide-2be3c335005b4bd28da38c62cf3e6a9b).

Reviewer: thirdweb is already a faucet listed on the user guide, right?

```suggestion
1. Get Scroll Sepolia ETH from any of the [Scroll Sepolia faucets](https://docs.scroll.io/en/user-guide/faucet/) listed in our docs. To build with L1SLOAD, get Scroll Devnet token by following [the L1SLOAD guide](https://scrollzkp.notion.site/L1SLOAD-Guide-2be3c335005b4bd28da38c62cf3e6a9b).
```

2. Inside the `packages/hardhat` directory, copy `.env.example` to `.env`.

```bash
cd packages/hardhat && cp .env.example .env
```

> 3. Edit your `.env` to specify the environment variables. Only specifying the `DEPLOYER_PRIVATE_KEY` is necessary here. The contract will be deployed from the address associated with this private key, so make sure it has enough Sepolia ETH.

Reviewer: ```suggestion
3. Edit your `.env` to specify the environment variables. Only specifying the `DEPLOYER_PRIVATE_KEY` is necessary here. The contract will be deployed from the address associated with this private key, so make sure it has enough Scroll Sepolia ETH or Scroll Devnet ETH depending on which network you are deploying on.
```

> 4. Use command to deploy all of your contracts:

Reviewer: ```suggestion
4. Use these commands to deploy all of your contracts:
```

  
```
# for scroll sepolia
yarn deploy --network scrollSepolia
# for L1SLOAD

> yarn deploy --network l1SLOAD

Reviewer: ```suggestion
yarn deploy --network L1SLOAD
```

```

**What's next**:

- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
- Edit your deployment scripts in `packages/hardhat/deploy`
- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`

## Adding Foundry

To add Foundry, follow this simple [tutorial](https://hardhat.org/hardhat-runner/docs/advanced/hardhat-and-foundry) by Hardhat.

## Documentation

> Visit [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2 which forked by Scaffold-Scroll.

Reviewer: ```suggestion
Visit the Scaffold-Eth [docs](https://docs.scaffoldeth.io) to learn how to start building with by Scaffold-Scroll, a fork of Scaffold-Eth-2.
```
